docs/RELEASE.md

```diff
@@ -0,0 +1,46 @@
+# Release cycle and versioning
+
+## Versioning
+Versioning is using SemVer (X.Y.Z):
+- The X is a **major version** and is incremented when:
+    - Support for older Zabbix versions is removed.
+    - Support for older Ansible versions is removed.
+    - Support for older Python versions is removed.
+    - Modules, roles or plugins are removed.
+    - Module or role functionality is removed.
+    - Other breaking changes or backward-incompatible changes are introduced.
+
+- The Y is a **minor version** and is incremented when:
```

Comment:
From our comments: "This is the main number we are focusing on and only do Z releases if we need to fix something quickly (e.g. right after we release Y and find bug"